R/estimate_KM.R

```diff
@@ -65,13 +68,14 @@ estimate_KM <- function(
  ### Magrittre pipe returns "." which inactivates recalls to survfit in downstream functions
  ### map passes .x as Call$data
  ### df: catch expressions that represent base R subsets
-  
   Call <- as.list(match.call())
   dots <- list(...)
   dfExpr <- Call[["data"]]
   
  ## Validate `data` and capture data name
  
+  reqcols <- c(strata, CNSR, AVAL)
```

Comment:
move back to validate columns section

Reply:
How do you mean, move back?

Reply:
This part of the code was under the next point ## validate columns

Reply:
Ah, thank you!